fingerprintjs/fingerprintjs-pro-react-native
# Add hooks approach

- Added hooks approach
- Tested on android
- Added unit tests
- Check types
- Github action for build, tests, and types
- Add linter
